.github/scripts/comment_on_pr.py

```diff
@@ -74,7 +74,7 @@ def check_code_changes(before_code, after_code):
         "top_p": 0.7,
```

Comment:
Your code review request has been updated to be more specific and concise. Here's a review of the changes:

### Before
- The system message was somewhat vague, asking for a comment on code changes without clearly defining what kind of feedback is expected.
- The user message had several issues:
  - The `top_p`, `temperature`, `max_tokens`, and `stop` parameters were incorrectly indented and placed within the user message.
  - The `stop` parameter values were not properly formatted.

### After
- The system message is clearer, asking for advice on code improvements.
- The user message has been corrected:
  - The `top_p`, `temperature`, `max_tokens`, and `stop` parameters are now correctly indented and placed within the user message.
  - The `stop` parameter values are properly formatted.

### Advice for Improvement
1. **System Message Clarity**: While the updated system message is clearer, consider providing more context or examples of what constitutes an "area that needs improvement" to ensure the model generates helpful feedback.
2. **User Message Structure**: Ensure that all parameters related to the user's request are correctly formatted and indented to maintain readability and avoid syntax errors.
3. **Parameter Values**: Verify that the `stop` parameter values are appropriate for your use case. The current values seem to be placeholders (`"`, `"`, `"`, `"`) and should be replaced with actual stopping sequences if needed.

### Suggested Code Review Request
Here's a revised version of your code review request:

```json
{
    "messages": [
        {
            "role": "system",
            "content": "Please do the code review for changes below. If there are any areas that need improvement, please provide specific advice on how to enhance the code. If there are no issues, please return an explanation of the change."
        },
        {
            "role": "user",
            "content": "Here are the code changes:",
            "parameters": {
                "top_p": 0.7,
                "temperature": 0.9,
                "max_tokens": 1024,
                "stop": ["", "", "", ""]
            }
        }
    ]
}
```

Make sure to replace the placeholder `"Here are the code changes:"` with the actual code changes you want to review.